graphql-server/migrations/1589049353064_alter_table_public_games_add_column_allow_card_skips/down.yaml

```diff
@@ -0,0 +1,5 @@
+- args:
+    cascade: false
+    read_only: false
+    sql: ALTER TABLE "public"."games" DROP COLUMN "allow_card_skips";
+  type: run_sql
```

Comment:
For future - it's helpful to split up migrations from app code changes

I haven't yet automated applying migrations on deploy (it's possible), so there's a short window for me to apply migrations (and ensure there's no issues) before then the app code also getting released and depends on them to be there.

Also there's a query white list (viewable at https://fishbowl-graphql.onrender.com/console/settings/allowed-queries). Each time we update any *.graphql file, the order of operations is:

1. Before merging code, set `HASURA_GRAPHQL_ENABLE_ALLOWLIST` to false at https://dashboard.render.com/web/srv-bqave7tp1qr5voljem1g/env
2. Deploy the code + update the query/mutation(s) respectively w/the new code.
3. After the FE is deployed w/the new code, set `HASURA_GRAPHQL_ENABLE_ALLOWLIST` back to true.

^ the root of this issue is that we don't control deploy order. Ideally we'd stand up the entire infra, update the query whitelist, etc. And then do a transition of new FE/new BE with the old FE/old BE, draining old requests.

I can do these rlly fast now (from practice haha), so no worries this time around.

cc: @wtachau @jbrambleDC for awareness!

Related infra stories
- https://trello.com/c/4TEfGCPT/92-auto-apply-migrations-on-docker-start-up-in-render

Reply:
after all that... I disabled the allow list lol. it's causing more pain than the benefit is worth.. and more challenging in config v2. per https://github.com/avimoondra/fishbowl/pull/29